### DISCUSSION TITLE
Vignette example reproduction question

### DISCUSSION THREAD

**Discussion by him1532** [satijalab/seurat#6307] [Q&A]
Hi I am complete novice and have been following vignette examples.
When following the example code on  'Using seurat with multimodal data' section  
following code gave an error saying there is NA in the data.

`cbmc<-FindVariableFeatures(cbmc)`
`Error in hvf.info$variance.expected[not.const] <- 10^fit$fitted : 
NAs are not allowed in subscripted assignments`

After googling I removed the gene  that had NA  with following code 

`which(rowSums(is.na(cbmc@assays$RNA@counts))>0)` 
`RKeep<-rownames(cbmc)[!rownames(cbmc) %in% "DNMT3A"] `
`cbmc<-subset(cbmc, features = RKeep)`

But then after that the Dimplot, UMAP plot, VlnPlot all show a different plot than
the example given on the vignette.
Is this expected result?

Thank you.

**Comment by thesnowdream** [comment]
Hello, I wonder if you have solved this problem and I have met the same one. I searched the Internet but still cannot solve it. Thanks for your help!
